main.go

```diff
@@ -120,6 +120,17 @@ func main() {
 		ReportOnly:           false,
 	}).Methods("POST")
 
+	r.HandleFunc("/report/csp", handler.ReportAPICorsHandler).Methods("OPTIONS")
```

Comment:
same note as the README on moving the path to avoid confusion.

Reply:
agreed